templates/seL4VMParameters.template.c

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright 2023, DornerWorks
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <camkes.h>
+#include <arm_vm/gen_config.h>
+
+/*- if configuration[me.name].get('vm_address_config') -*/
```

Comment:
Can we cache `configuration[me.name].get('vm_address_config')` in a script variable, so this becomes a bit easier to red and less redundant?

Reply:
Sure thing.